pkgs/data/fonts/readexpro/default.nix

```diff
@@ -0,0 +1,49 @@
+{
+  pkgs ? import <nixpkgs> { },
+  lib ? import <nixpkgs/lib>,
+  fetchgit ? pkgs.fetchgit,
+  stdenvNoCC ? pkgs.stdenvNoCC,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "readexpro";
+  version = "1.2";
+
+  src = fetchgit {
+    url = "https://github.com/ThomasJockin/readexpro.git";
```

Comment:
We usually prefer using `fetchFromGitHub` for Git repositories hosted on GitHub, since we can download the entire repo as a tarball instead of needing to use `git clone` (which is much slower). Are the extra files in the repo so big that it's worth to use a sparse checkout here?